lib/web_ui/lib/src/engine/canvas_pool.dart

```diff
@@ -520,6 +520,11 @@ class _CanvasPool extends _SaveStackTracking {
             break;
           case PathCommandTypes.ellipse:
             final Ellipse ellipse = command;
+            if (c == 0) {
+              // Ellipses that start a new path need to set start point,
+              // otherwise it incorrectly uses last point.
+              ctx.moveTo(subpath.startX, subpath.startY);
+            }
```

Comment:
Is this not the case for other path command types?

Reply:
nvm, all other commands assume you are already at some point, and they draw from there.